projects/packages/import/src/endpoints/trait-import.php

```diff
@@ -12,6 +12,11 @@
 /**
  * Import trait. Base class for all import endpoints.
  */
+/**
+ * Batching details.
+ *
+ * @property array|false $allow_batch
+ */
```

Comment:
This confuses the phpdoc, the doc comment for the class is now "Batching details".
```suggestion
 *
 * @property array|false $allow_batch Batching details.
 */
```

Reply:
PHPCS complains about that format with `Generic.Commenting.DocComment.MissingShort`. Moving it inside the class doesn't satisfy Phan. It does look like `//` would work in that format; does that confuse PHPDoc?

Reply:
Ah, got it. ef6578b4ba

Reply:
PHPDoc only pays attention to `/**` comments, I believe, so `//` would be fine there. OTOH, I don't know which kinds of comments Phan pays attention to for `@property`.